shear_meas_test/test_shear_meas.py

```diff
@@ -146,6 +146,132 @@ def make_sim(
     return mbobs
 
 
+def make_sim_color(
+    *,
+    seed,
+    g1,
+    g2,
+    dim=251,
+    buff=34,
+    scale=0.25,
+    dens=100,
+    ngrid=7,
+    snr=1e6,
+):
+    rng = np.random.RandomState(seed=seed)
+
+    half_loc = (dim-buff*2)*scale/2
+
+    if ngrid is None:
+        area_arcmin2 = ((dim - buff*2)*scale/60)**2
+        nobj = int(dens * area_arcmin2)
+        x = rng.uniform(low=-half_loc, high=half_loc, size=nobj)
+        y = rng.uniform(low=-half_loc, high=half_loc, size=nobj)
+    else:
+        half_ngrid = (ngrid-1)/2
+        x, y = np.meshgrid(np.arange(ngrid), np.arange(ngrid))
+        x = (x.ravel() - half_ngrid)/half_ngrid * half_loc
+        y = (y.ravel() - half_ngrid)/half_ngrid * half_loc
+        nobj = x.shape[0]
+
+    cen = (dim-1)/2
+    psf_dim = 53
+    psf_cen = (psf_dim-1)/2
+    us = rng.uniform(low=-scale, high=scale, size=nobj)
+    vs = rng.uniform(low=-scale, high=scale, size=nobj)
+    colors = rng.randint(low=0, high=2, size=nobj)
+
+    mbobs = ngmix.MultiBandObsList()
+    base_psf = galsim.Gaussian(fwhm=0.9)
+    # this dict holds the PSFs as a function of color (c0 or c1) and band (index 0 or 1)
+    psfs = {
+        "c0": [galsim.Gaussian(fwhm=0.8), galsim.Gaussian(fwhm=0.95)],
+        "c1": [galsim.Gaussian(fwhm=0.85), galsim.Gaussian(fwhm=1.0)],
```

Comment:
I'm a little concerned that these are all symmetrically arranged around the base_psf.  So errors could be canceling over the full set.
Maybe make sure that when selecting the subset of galaxies in each color or just using one band, you still get the right answer (m=0).